tools/ci_build/github/azure-pipelines/templates/py-packaging-linux-test.yml

```diff
@@ -0,0 +1,60 @@
+parameters:
+- name: arch
+  type: string
+
+- name: device
+  type: string
+
+- name: machine_pool
+  type: string
+
+- name: extra_job_id
+  type: string
+  default: ''
+
+- name: python_wheel_suffix
+  type: string
+  default: ''
+
+jobs:
+- job: Linux_Test_${{ parameters.device }}${{ parameters.extra_job_id }}_${{ parameters.arch }}
+  timeoutInMinutes: 240
+  variables:
+    skipComponentGovernanceDetection: true
+  workspace:
+    clean: all
+  pool: ${{ parameters.machine_pool }}
+  steps:
+  - task: DownloadPipelineArtifact@2
+    displayName: 'Download Pipeline Artifact'
+    inputs:
+      preferTriggeringPipeline: true      
+      artifactName: 'drop-linux-${{ lower(parameters.device) }}-${{ parameters.arch }}'
+      targetPath: '$(Build.BinariesDirectory)/Release'
+      runVersion: 'latest'
+      buildType: specific
```

Comment:
I set preferTriggeringPipeline to true. Its doc says: "If checked, this build task will try to download artifacts from the triggering build. If there is no triggering build from the specified pipeline, it will download artifacts from the build specified in the options below". So far from my observations, it works as expected. If later on we found it has problems, we can do it in a way like:
```
- task: DownloadPipelineArtifact@2
  inputs:
    source: 'specific'
    artifact: 'drop'
    path: $(Build.SourcesDirectory)/bin
    project: 'xxxx'
    pipeline: 000000
    runVersion: 'specific'
    runId: $(resources.build.runid)
```